python/pyspark/sql/conf.py

```diff
@@ -39,7 +40,7 @@ def set(self, key: str, value: str) -> None:
         self._jconf.set(key, value)
 
     @since(2.0)
-    def get(self, key: str, default: Optional[str] = _NoValue) -> str:
+    def get(self, key: str, default: Union[Optional[str], _NoValueType] = _NoValue) -> str:
```

Comment:
files in `pyspark/sql` need to be changed because mypy check type of `_NoValue` after we inline type hint for `pyspark/__init__.py`